src/test/java/com/iexec/standalone/result/ResultServiceTests.java

```diff
@@ -72,39 +68,45 @@
 import static com.iexec.standalone.task.TaskTestsUtils.getStubTask;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.*;
 import static org.mockito.Mockito.*;
-import static org.mockito.Mockito.times;
 
-@Slf4j
+//@Slf4j
 class ResultServiceTests {
 
     public static final String RESULT_DIGEST = "0x0000000000000000000000000000000000000000000000000000000000000001";
     // 32 + 32 + 1 = 65 bytes
     public static final String ENCLAVE_SIGNATURE = "0x000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000b0c";
-    private static final String CHAIN_TASK_ID = "taskId";
+    private static final String CHAIN_DEAL_ID = "0xe1f3b96f58be8d5d1958ac14b6a3e93497ad9985ea44ac8c79f613129fff79a0";
+    private static final String CHAIN_TASK_ID = "0x7602291763f60943833c39a11b7e81f1f372f29b102bffad5b23c62bde0ef70e";
     private static final String CHAIN_TASK_ID_2 = "taskId2";
     private static final String IEXEC_WORKER_TMP_FOLDER = "src"
-                                                        + File.separator + "test"
-                                                        + File.separator + "resources"
-                                                        + File.separator + "tmp"
-                                                        + File.separator + "test-worker"
-                                                        + File.separator;
+            + File.separator + "test"
+            + File.separator + "resources"
+            + File.separator + "tmp"
+            + File.separator + "test-worker"
+            + File.separator;
```

Comment:
Is this due to Windows compatibility considerations ?

Reply:
Yes it is